packages/promise-kit/index.js

```diff
@@ -3,17 +3,26 @@
 
 /// <reference types="ses"/>
 
+import { memoRace } from './memo-race.js';
+
 /** @type {PromiseConstructor} */
 const BestPipelinablePromise = globalThis.HandledPromise || Promise;
 
 /**
  * @template T
- * @typedef {Object} PromiseKit A reified Promise
+ * @typedef {object} PromiseKit A reified Promise
  * @property {(value: ERef<T>) => void} resolve
  * @property {(reason: any) => void} reject
  * @property {Promise<T>} promise
  */
 
+/**
+ * @template T
+ * @callback PromiseExecutor The promise executor
```

Comment:
How aware is the promiseKit package of eventual-send / HandledPromise? Should the executor type include `resolveToPresence`?

Reply:
It is not and ignoring it, like the existing `makePromiseKit`